initpost.sh

```diff
@@ -43,10 +44,20 @@ FILE_NAME="${CURRENT_DATE}-${POST_NAME}.md"
 # Set your destination folder
 BINPATH=$(cd `dirname $0`; pwd)
 POSTPATH="${BINPATH}/_posts"
-DIST_FOLDER="$POSTPATH"
+DRAFTPATH="${BINPATH}/_drafts"
+
+if [[ "${1}" == "-c" || "${1}" == "--create" ]]; then
+    DIST_FOLDER="$POSTPATH"
+    FILE_NAME="${CURRENT_DATE}-${POST_NAME}.md"
+fi
+
+if [[ "${1}" == "-d" || "${1}" == "--draft" ]]; then
+    DIST_FOLDER="$DRAFTPATH"
+    FILE_NAME="${POST_NAME}.md"
+fi
 
 # Set your blog URL
-BLOG_URL="http://willianjusten.com.br"
+BLOG_URL="http://vsential.com"
```

Comment:
Please, it was my mistake, change this to something like "http://yourblogurl.com", it's more generic than our url...